BEIMA.Client/src/pages/Users/UserPage.js

```diff
@@ -91,13 +76,26 @@ const UserPage = () => {
       }
 
       // Call Update User
-      console.log(newUser);
-      setEditable(false)
+      let updateResult = await UpdateUser(newUser);
+      if(updateResult.status === Constants.HTTP_SUCCESS){
+        Notifications.success("Update User Successful", `User ${username} updated successfully.`);
+        setEditable(false);
+      } else {
+        Notifications.error("Unable to Update User", `Update of User ${username} failed.`);
+      }
     }
     
-    const deleteUserCall = () => {
-      //TODO integrate with styled error/confirm message
-      alert("you sure?");
+    const deleteUserCall = async () => {
+      let deleteNotif = await Notifications.warning("Warning: User Deletion", [`Are you sure you want to delete User ${username}?`]);
+      if(deleteNotif.isConfirmed){
+        let deleteResult = await DeleteUser(userId);
+        if(deleteResult.status === 200){
+          Notifications.success("User Deletion Successful", `User ${username} successfully deleted.`);
+          navigate('/users');
+        } else {
+          Notifications.error("Unable to Delete User", `Deletion of User ${username} failed.`);
```

Comment:
I can update this to use the message from the endpoint in the case of an admin is trying to delete the only admin account

Reply:
I think that would be a good idea, just to be more specific about the error. We could even make a bunch of else ifs instead if we go based off of the status code and/or message and customize our messages for each of those. But yeah, just relaying the message in the response would be the simplest way to go.